README.md

```diff
@@ -168,6 +168,7 @@ and start it:
 
 ```shell
 $ ./bin/ocagent_$(go env GOOS)
+$ 2018/10/08 21:38:00 Running OpenCensus interceptor as a gRPC service at "127.0.0.1:55678"
```

Comment:
s/interceptor/receiver/g as it has just changed 2 PRs ago

Reply:
good catch